src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -271,6 +271,16 @@ class ResourceService(
         }
       )
 
+  def setResourceAuthDomain(
```

Comment:
This function should validate the auth domains probably using `validateAuthDomain` function